schemas/service.py

```diff
@@ -0,0 +1,16 @@
+from pydantic import BaseModel
+from typing import List, Optional
+from datetime import datetime
+
+class Resource(BaseModel):
+    name: str
+    status: str
```

Comment:
We should be trying to specify values where possible, e.g. `status` should probably be a `Literal`: https://docs.pydantic.dev/latest/api/standard_library_types/#typingliteral